setup.py

```diff
@@ -190,6 +191,11 @@ def run(self):
             elif os.path.isdir(f):
                 shutil.rmtree(f)
 
+
+        for name in glob.glob(dir_setup+"/doc/src/modules/physics/vector/*.pdf"):
```

Comment:
It is best to use `os.path.join` to build up path strings in a cross platform way.